client/src/pages/yourorders/YourOrders.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import {useAuth} from "../../contexts/AuthContext";
+
+import axios from "axios";
+import OrderCard from "./components/orderCard";
+import BtnSpinner from "../../utils/BtnSpinner";
+
+const MyOrders = () => {
+    const {uid} = useAuth();
+    const [orders,setOrders] = useState([]);
+    const [spinner,setSpinner] = useState(false);
+
+    useEffect( () => {
+        ( async function(){
+          setSpinner(true)
+          try {
+            const res = await axios.get("https://shopping-hub-2021.herokuapp.com/api/orders/"+uid);
```

Comment:
Suggestion: template literal would be more readable.

Reply:
yeah will add templete literals and refactored async calls code later